PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+
+/// PiwikUserDefaults is a wrapper for the UserDefaults with properties
+/// mapping onto values stored in the UserDefaults.
+/// All getter and setter are sideeffect free and automatically syncronize
+/// after writing.
+internal class PiwikUserDefaults {
+    internal static let standard = PiwikUserDefaults()
```

Comment:
I would not use any access modifiers for members unless they should have a more restrictive modifier then the enclosing type. Less clutter and typing.
Style question, up to you.